packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -61,9 +69,26 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
   }
 
   public componentDidMount() {
-    const { initialScrollPosition } = this.props;
-    this._events.on(this.contentContainer, 'scroll', this._onScroll);
-    this._events.on(window, 'resize', this._onWindowResize);
+    const { initialScrollPosition, scrollbarVisibility } = this.props;
+    if (scrollbarVisibility === ScrollbarVisibility.always) {
+      // after first render, scrollbars are visible
+      // it is needed to position stickyContainers correctly
+      let scrollbarHeight: number | undefined = this._getScrollbarHeightFromLocalStorage();
+      if (!scrollbarHeight) {
+        scrollbarHeight = this._getScrollbarHeight();
+        this._storeScrollbarHeightInLocalStorage(scrollbarHeight);
```

Comment:
This seems wrong. Why would a UI component cache the height of a single instance in localstorage? What if there is more than one on the page?

Reply:
The height of scrollbar refers to the height of horizontal scrollbar. It remains same for all the instances on a page. Also, the height of horizontal scrollbar is same as the width of vertical scrollbar.